decidim-accountability/app/views/decidim/accountability/admin/results/index.html.erb

```diff
@@ -4,6 +4,7 @@
       <%= link_to "#{translated_attribute(parent_result.title)} > ", edit_result_path(parent_result) if parent_result %>
       <%= t(".title") %>
       <div class="button--title">
+        <%= link_to t("actions.import", scope: "decidim.accountability", name: t("models.result.name", scope: "decidim.accountability.admin")), new_projects_import_path, class: "button tiny button--simple" if allowed_to? :create, :result unless parent_result %>
```

Comment:
Avoid using `if a ... unless b` with one liners.

Instead use `if a && !b`, it's more readable in this context.

I also think this permission check should be the same as you are using in the controller to check whether the user can access those actions?